buildDocs.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+const themes = require('./dist/index.js')
+
+const colorKeys = ['Color', 'BackgroundColor', 'BorderColor', 'TextColor']
+
+const stream = fs.createWriteStream("docs/tokens.md", {flags:'a'});
+Object.keys(themes.light).forEach(key => {
+  stream.write(`## ${key}\n`)
+  if (colorKeys.indexOf(key) !== -1) {
+    stream.write(`
+| Key | Value | Color |
+| --- | ----- | ----- |
+`)
+  } else {
+    stream.write(`
+| Key | Value |
+| --- | ----- |
+`)
+  }
+
+  Object.keys(themes.light[key]).forEach(childKey => {
+    const value = themes.light[key][childKey]
+
+    if (colorKeys.indexOf(key) !== -1) {
+      const hash = value.indexOf('#') !== -1
+
+      if (value.indexOf('#') !== -1) {
+        stream.write(`| ${childKey} | ${value} | ![${value}](https://placehold.it/15/${value.split('#')[1]}/000000?text=+) |\n`)
```

Comment:
This is neat! I was thinking we might need a script to generate an image of the different colors for documentation. I am glad there's a simpler solution.